mods/ts/rules/nod-support.yaml

```diff
@@ -106,6 +106,9 @@ NAFNCE:
 	EnergyWall:
 		ActiveCondition: active-posts == 2
 		Weapon: LaserFence
+		TerrainTypes: Clear, Rough, Road, DirtRoad, Green, Sand, Pavement
+	RequiresBuildableArea:
+		Adjacent: 4
```

Comment:
Yes, the Post has Adjacency value of 4, old code was using 4 here too since it read from the Post, so this is to keep the old behaviour. Using 7 would allow it to connect 2 disconnected buildable areas that 4 otherwise wouldn't, but i think them matching is better here.

Tho, i'm not sure how it worked in the original game, if someone check the original behaviour we can change it to match.